src/main/java/network/brightspots/rcv/GuiConfigController.java

```diff
@@ -307,20 +307,21 @@ private String getOvervoteRuleChoice() {
    * Action when help menu item is clicked. Try to open the local help manual.
    */
   public void menuItemOpenHelpClicked() {
-    URL helpFileUrl = ClassLoader.getSystemResource(CONFIG_FILE_DOCUMENTATION_FILENAME);
+    String helpFile = ClassLoader.getSystemResource(CONFIG_FILE_DOCUMENTATION_FILENAME).getFile();
     String command = null;
     if (Utils.IS_OS_WINDOWS) {
-      command = String.format("cmd /c start \"Help\" \"%s\"", helpFileUrl);
+      command = String.format("cmd /c start \"Help\" \"%s\"", helpFile);
     } else if (Utils.IS_OS_MAC) {
-      command = String.format("open %s", helpFileUrl);
+      command = String.format("/usr/bin/open %s", helpFile);
     } else if (Utils.IS_OS_LINUX) {
-      command = String.format("xdg-open \"%s\"", helpFileUrl);
+      command = String.format("xdg-open \"%s\"", helpFile);
     } else {
       Logger.info("Unable to determine operating system. Try opening the documentation "
-          + "manually at: %s", helpFileUrl);
+          + "manually at: %s", helpFile);
```

Comment:
So this won't work on my Amiga?

Reply:
No Newton either :/

Reply:
https://www.youtube.com/watch?v=u6qxixgQJ4M